packages/connectivity/connectivity_macos/pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: connectivity_macos
+description: Temporary desktop implmentation of connectivity from flutter/plugins
```

Comment:
Why is this temporary? I think this needs a README to explain what this is in more detail.

Reply:
It's not, I forgot to change that message :) Done